### PR TITLE
fix(docs): update auth cli link to link back to rekognition docs for setup

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Originally amplify auth link pointed to the ampliy cli docs, this was confusing as this was the general auth set up
- Rekognition docs has specific instructions to follow when setting up amplify auth so linking there instead - https://docs.aws.amazon.com/rekognition/latest/dg/face-liveness-prerequisites.html#face-liveness-prerequisites-amplify
- Also added a note for customers who want to not use amplify cli and import their own auth
- Added a section for manually configuring amplify auth

Screenshot 2023-04-26 at 4 58 09 PM
Screenshot 2023-04-26 at 4 58 04 PM

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by hbuchel — APPROVED** [review]

**Review by reesscot — DISMISSED** [review]

**Review by zchenwei — DISMISSED** [review]

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.android.mdx [hunk 1]
```diff
@@ -1,4 +1,6 @@
-FaceLivenessDetector uses Amplify Auth by default to authorize users to perform the Face Liveness check. [Configure the Amplify Auth category](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android/#configure-auth-category) to setup Amplify Auth and add authentication to your backend configuration.
+FaceLivenessDetector uses Amplify Auth by default to authorize users to perform the Face Liveness check. [Configure the Amplify Auth category](https://docs.aws.amazon.com/rekognition/latest/dg/face-liveness-prerequisites.html#face-liveness-prerequisites-amplify) to setup Amplify Auth and add authentication to your backend configuration.
+
+> If you already have Cognito set up or do not want to use the Amplify CLI to generate Cognito resources, you can follow the documentation for [importing existing resources](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android/#set-up-backend-resources).
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.android.mdx:3]: This link seems wrong?

Comment (reesscot) [reply docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.android.mdx:3]: I think it should be https://docs.amplify.aws/cli/auth/import/ ?

Comment (reesscot) [reply docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.android.mdx:3]: Oh dang, that's hard to find...

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 2]
```diff
@@ -1,5 +1,44 @@
...
+> Use the Amplify CLI to if you are interested in Amplify automatically configurating and managing your Cognito Identity and User Pool for you.
+
+FaceLivenessDetector uses Amplify Auth by default to authorize users to perform the Face Liveness check. If you are setting up Amplify for the first time, follow the instructions in the developer guide for [setting up Amplify Auth with our recommended settings](https://docs.aws.amazon.com/rekognition/latest/dg/face-liveness-prerequisites.html#face-liveness-prerequisites-amplify).
 
 _If you have an existing Amplify backend, run `amplify pull` to sync your `aws-exports.js` with your cloud backend._
 
-You should now have an `aws-exports.js` file in your `src` directory with your latest Amplify backend configuration.
+Once complete you should now have an `aws-exports.js` file in your `src` directory with your latest Amplify backend configuration.
+  </TabItem>
+  <TabItem title="Existing Resource">
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:15]: Maybe 
```suggestion
  <TabItem title="Create/Reuse Existing Cognito Resource">
```

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 3]
```diff
@@ -1,5 +1,44 @@
-FaceLivenessDetector uses Amplify Auth by default to authorize users to perform the Face Liveness check. [Configure the Amplify Auth category](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/#create-authentication-service) to setup Amplify Auth and add authentication to your backend configuration.
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+
+There are two methods to setup the Amazon Cognito resources needed for Amplify Auth. The most common is the Amplify CLI, which asks a series of questions and will configure both a User Pool and Identity Pool automatically. The second option is to reuse or create a Cognito Identity Pool manually and to import it into your application.
+
+<Tabs>
+  <TabItem title="Amplify CLI">
+> Use the Amplify CLI to if you are interested in Amplify automatically configurating and managing your Cognito Identity and User Pool for you.
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:7]:
```suggestion
> Use the Amplify CLI for Amplify to automatically configure and manage your Cognito Identity and User Pool for you.
```

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 4]
```diff
@@ -1,5 +1,44 @@
...
 
-You should now have an `aws-exports.js` file in your `src` directory with your latest Amplify backend configuration.
+Once complete you should now have an `aws-exports.js` file in your `src` directory with your latest Amplify backend configuration.
+  </TabItem>
+  <TabItem title="Existing Resource">
+> Use this option if you already have a Cognito identity/user pools, or want to manage Cognito resources yourself or with a 3rd party resource management tool.
+
+If you already have Cognito set up or do not want to use the Amplify CLI to generate Cognito resources, you can follow the documentation for [importing existing resources](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/#set-up-backend-resources) or check out the code snippet below.
+
+_If you are manually setting up an identity pool in the Cognito console you need to use the v1 Cognito console and visit the [Federated Identities](https://console.aws.amazon.com/cognito/federated) section. When setting up the identity pool ensure that acess to unauthenticated identities is enabled._
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:20]:
```suggestion
_If you are manually setting up an identity pool in the Cognito console you need to use the v1 Cognito console and visit the [Federated Identities](https://console.aws.amazon.com/cognito/federated) section. When setting up the identity pool ensure that access to unauthenticated identities is enabled._
```

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 5]
```diff
@@ -1,5 +1,44 @@
...
+
+```jsx
+import React from 'react';
+import { ThemeProvider } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+import '@aws-amplify/ui-react/styles.css';
+import awsexports from './aws-exports';
+
+Amplify.configure({
+  "aws_cognito_region": "us-east-2", // (required) - Region where Amazon Cognito project was created
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:32]: Do we need to address possible need for user pool? Is there any reason they might need that?

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx (hunk 4 above)
Comment (reesscot) [reply docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:20]: I think a follow up should be an example using CDK

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx (hunk 5 above)
Comment (thaddmt) [reply docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:32]: If they use cognito for their overall amplify authentication or if they only want liveness checks to occur behind the authenticator, in that case though they can refer to the import docs

**Review by zchenwei — APPROVED** [review]

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 6]
```diff
@@ -1,5 +1,44 @@
...
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:32]:
```suggestion
   ...awsexports,
  "aws_cognito_region": "us-east-2", // (required) - Region where Amazon Cognito project was created
```

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.android.mdx (hunk 1 above)
Comment (reesscot) [reply docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.android.mdx:3]: Can we mention looking for the "existing resources" tab?

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 7]
```diff
@@ -1,5 +1,52 @@
...
+> If you previously had unmanaged resources that you want to manage with Amplify you can use the CLI to import your Cognito resources.
+
+FaceLivenessDetector uses Amplify Auth by default to authorize users to perform the Face Liveness check. Follow the instructions for [Amplify CLI (Import)](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/#set-up-backend-resources) to manage your Cognito resources with Amplify.
+
+Once complete you should now have an `aws-exports.js` file in your `src` directory with your latest Amplify backend configuration.
+  </TabItem>
+  <TabItem title="Self Managed Cognito Resource">
+> Use this option if you already have a Cognito identity/user pools that you do not want to import to Amplify, or want to manage Cognito resources yourself or with a 3rd party resource management tool.
+
+If you already have Cognito set up or do not want to use the Amplify CLI to generate Cognito resources, you can follow the documentation for in the [existing resources tab](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/#set-up-backend-resources) or check out the code snippet below.
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:25]:
```suggestion
If you already have Cognito set up or do not want to use the Amplify CLI to generate Cognito resources, you can follow the documentation in the [existing resources tab](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/#set-up-backend-resources) or check out the code snippet below.
```

**Review by reesscot — APPROVED** [review]

**Review by calebpollman — APPROVED** [review]

**Pull request merged**